docs/reference/commandline/exec.md

```diff
@@ -39,6 +39,12 @@ The command started using `docker exec` only runs while the container's primary
 process (`PID 1`) is running, and it is not restarted if the container is
 restarted.
 
+> **Note**: COMMAND run using the last `WORKDIR` specified.
```

Comment:
Could we remove the `For UNIX systems, this is / and for Windows [I got no idea]. ` part?